.travis.yml

```diff
@@ -26,6 +27,7 @@ install:
         docker run --rm $DOCKER_REPO:$VERSION-$ARCH-$DIST uname -a;
     done
 after_success:
+  - bash <(curl -s https://copilot.blackducksoftware.com/ci/travis/scripts/upload)
```

Comment:
If it all works, these changes also need to be added to `update-travis-config.sh`.
Otherwise they will be lost next time someone runs `update.sh`